Week-5/W5Exercise5.4/index.test.js

```diff
@@ -0,0 +1,17 @@
+const mathOps = require("./index");
+
+describe("Math operations testing", () => {
```

Comment:
could have tested existence of functions (sum, diff, product)

Reply:
I'll keep that in mind